### PR TITLE
Add new drawables and update drawable refrences

### PR DESCRIPTION
**Pull request by aminsato**
This commit includes the additions of new PNG assets for various chians and update corresponding part of codebase to these news assets

### CODE REVIEW

File: app/src/main/java/com/vultisig/wallet/models/Chain.kt [hunk 1]
```diff
@@ -125,11 +125,11 @@ val Chain.logo: Int
         Chain.solana -> R.drawable.solana
         Chain.ethereum -> R.drawable.ethereum
         Chain.avalanche -> R.drawable.avax
-        Chain.base -> R.drawable.eth_base
-        Chain.blast -> R.drawable.eth_blast
-        Chain.arbitrum -> R.drawable.eth_arbitrum
-        Chain.polygon -> R.drawable.eth_polygon
-        Chain.optimism -> R.drawable.eth_optimism
```
Comment (YGolybev) [inline app/src/main/java/com/vultisig/wallet/models/Chain.kt:132]: Are old resources still in use after this PR? If not they should probably be deleted

**Review by YGolybev — COMMENTED** [review]
Please, upload assets using Webp where possible

File: app/src/main/java/com/vultisig/wallet/models/Chain.kt (hunk 1 above)
Comment (aminsato) [reply app/src/main/java/com/vultisig/wallet/models/Chain.kt:132]: Yes
like https://github.com/vultisig/vultisig-android/blob/d42af0710fa2fc618ee75656cf260add3aacaf9e/app/src/main/java/com/vultisig/wallet/models/Coin.kt#L288

**Comment by aminsato** [comment]
> Please, upload assets using Webp where possible

**Comment by aminsato** [comment]
> Please, upload assets using Webp where possible

Can I upload .xml instead of Webp?

**Review by YGolybev — COMMENTED** [review]
If XML seems like a better choice for some assets. But I think that all of these assets will be problematic for XML.

**Comment by aminsato** [comment]
> If XML seems like a better choice for some assets. But I think that all of these assets will be problematic for XML.

Done

**Review by YGolybev — APPROVED** [review]

**Pull request merged**